repository/c.json

```diff
@@ -131,6 +131,16 @@
 			]
 		},
 		{
+			"name": "Cadence Skill",
+			"details": "https://github.com/leoheck/sublime-cadence-skill",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": true
```

Comment:
Your [current tag](https://github.com/leoheck/sublime-cadence-skill/tags) is not a valid [semver](http://semver.org).